postgresql/initdb.d/02_functions.sql

```diff
@@ -57,10 +57,10 @@ END;
 
 $set_archived$ LANGUAGE plpgsql;
 
-CREATE FUNCTION set_verified(file_uuid UUID, corr_id UUID, archive_checksum TEXT, archive_checksum_type TEXT, descrypted_size BIGINT, decrypted_checksum TEXT, decrypted_checksum_type TEXT)
+CREATE FUNCTION set_verified(file_uuid UUID, corr_id UUID, archive_checksum TEXT, archive_checksum_type TEXT, decrypted_size BIGINT, decrypted_checksum TEXT, decrypted_checksum_type TEXT)
 RETURNS void AS $set_verified$
 BEGIN
-    UPDATE sda.files SET decrypted_file_size = descrypted_size WHERE id = file_uuid;
+    UPDATE sda.files SET decrypted_file_size = decrypted_size WHERE id = file_uuid;
```

Comment:
Although you mentioned it's a typo fix for `descrypted_size`, I hope this change will not affect any other places.

Reply:
If you look at whats happening in the affected code block you will have your answer.